tests/test_autonomy/test_chain/test_addresses.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2023 Valory AG
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""Verify the on-chain addresses against the defined constants."""
+
+import typing as t
+
+import pytest
+import requests
+
+from autonomy.chain import constants
+from autonomy.chain.config import ChainType
+
+
+ADDRESS_FILE_URL = "https://raw.githubusercontent.com/valory-xyz/autonolas-registries/main/docs/configuration.json"
+CHAIN_SLUGS = {ChainType.ETHEREUM: "mainnet", ChainType.GOERLI: "goerli"}
```

Comment:
Please make a note somehow to account for other chains once they are available in the app.

Reply:
I am confused. Where are the tests for Gnosis and Polygon? The CLI should support them already no?

Reply:
They're supported via the ENV vars